screens/HomeScreen.tsx

```diff
@@ -186,6 +187,10 @@ export default function HomeScreen({ navigation, route }) {
 
   return (
     <SafeAreaView style={[styles.outer]}>
+      <AppDevAnnouncements
+        host={process.env.ANNOUNCEMENTS_HOST}
+        appPath={process.env.ANNOUNCEMENTS_PATH}
+      />
```

Comment:
The convention for this file is to have all modals be at the bottom of the `SafeAreaView`, let's follow that convention to keep the code readable